install.sh

```diff
@@ -8,28 +8,30 @@
 # This script will create links for the dotfiles.
 
 # Step 1: ensure all necessary directories exist before linking files.
-for dir in $(find dotfiles/ -mindepth 1 -type d); do
+for dir in $(ls | find * -mindepth 1 -type d ! -name '.git' ! -name '.gitignore'); do
```

Comment:
Why pipe the output of `ls` to `find`?
`find` won't do anything with it, it just ignores it.

Reply:
Not that it will take the output of the ls to the be the input of the find command.